### PR TITLE
Protect against dangerous default value.

### LINKED ISSUE #240: Pylint: W0102
```bash
mmpose/models/backbones/hrnet.py:19:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/hrnet.py:264:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:45:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:45:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:177:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:177:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/cpm.py:71:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/regnet.py:84:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/hourglass.py:26:4: ```: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/hourglass.py:115:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:34:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:149:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:327:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:477:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v3.py:65:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:32:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:32:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:126:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:126:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:31:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:31:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:153:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:153:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/scnet.py:25:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/utils/inverted_residual.py:36:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/utils/inverted_residual.py:36:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
```

### PR DESCRIPTION
**Pull request by yaochaorui**
Fix #240

### CODE REVIEW

File: mmpose/models/backbones/cpm.py [hunk 1]
```diff
@@ -75,6 +77,7 @@ def __init__(self,
                  middle_channels=32,
                  num_stages=6,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        norm_cfg = copy.deepcopy(norm_cfg)
```
Comment (innerlee) [inline mmpose/models/backbones/cpm.py:80]: need to add that line of comment, so that future reader know what this line is for

Comment (yaochaorui) [reply mmpose/models/backbones/cpm.py:80]: Apply to all?

Comment (innerlee) [reply mmpose/models/backbones/cpm.py:80]: yea

Comment (innerlee) [reply mmpose/models/backbones/cpm.py:80]: like this

image

Comment (yaochaorui) [reply mmpose/models/backbones/cpm.py:80]: Yes I see.

**Review by innerlee — APPROVED** [review]

File: mmpose/models/backbones/cpm.py [hunk 2]
```diff
@@ -75,6 +77,8 @@ def __init__(self,
                  middle_channels=32,
                  num_stages=6,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        # Protect against dangerous default value.
```
Comment (innerlee) [inline mmpose/models/backbones/cpm.py:80]: Use 
`Protect mutable default arguments`

Comment (innerlee) [reply mmpose/models/backbones/cpm.py:80]: The default values are not "dangerous" without knowing the context

**Review by innerlee — APPROVED** [review]
